qbx_consumables/client.lua

```diff
@@ -129,20 +129,23 @@ end
 exports('CokeBaggyEffect', cokeBaggyEffect)
 
 local function smokeWeed()
-    CreateThread(function()
-        while smokingWeed do
-            Wait(10000)
-            TriggerServerEvent('hud:server:RelieveStress', math.random(15, 18))
-            relieveCount += 1
-            if relieveCount == 6 then
-                exports.scully_emotemenu:cancelEmote()
-                if smokingWeed then
-                    smokingWeed = false
-                    relieveCount = 0
+    if not smokingWeed then
```

Comment:
```suggestion
    if smokingWeed then return end
```

and remove end from line 148